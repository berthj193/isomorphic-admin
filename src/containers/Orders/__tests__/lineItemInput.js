import React from 'react';
import LineInputItem from '../lineItemInput';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<LineInputItem />', () => {
  const defaultProps = {
    meta: {},
  };

  const getComponent = (props = {}) => shallow(<LineInputItem {...{ ...defaultProps, ...props }} />);

  it('should render properly', () => {
    expect(toJson(getComponent({ input: { onChange: () => null } }))).toMatchSnapshot();
  });
});
