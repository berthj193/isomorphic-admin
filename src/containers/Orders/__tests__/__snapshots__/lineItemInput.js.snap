// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LineInputItem /> should render properly 1`] = `
<FieldList
  addFieldLabel="Add item"
  component={[Function]}
  defaultItemValue=""
  itemClassName="field-list-input"
  label="Items"
  meta={Object {}}
  onChange={[Function]}
  value={Array []}
/>
`;
